schema/crdb/region-read-only/up02.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE omicron.public.region
+  ALTER COLUMN read_only DROP DEFAULT;
```

Comment:
Will this turn off the setting of a default value for the `read_only` column, meaning it must
be provided for any new insertions into the table?

Reply:
Yep